test/test_transforms_v2_refactored.py

```diff
@@ -228,26 +228,37 @@ def check_functional_kernel_signature_match(functional, *, kernel, input_type):
         assert functional_param == kernel_param
 
 
-def _check_transform_v1_compatibility(transform, input):
+def _check_transform_v1_compatibility(v2_transform_eager, input, rtol, atol):
```

Comment:
Now that we don't check against scripted v1, I think we can drop the `_eager` suffix.